packages/core/lib/Configuration.ts

```diff
@@ -19,6 +19,7 @@
 import Yargs = require("yargs");
 import * as path from "path";
 import shell = require("shelljs");
+import { singleTonAlreadySet } from "./Diagnostics";
```

Comment:
Why does `Singleton` have a capital `T` in the middle?

Reply:
Fixed.